File: src/consts.rs
pub const PYTHON_VENV_PATH: &str = ".pyenvs";
pub const PY_VENV_CFG: &str = "pyvenv.cfg";
#[cfg(target_family = "unix")]
pub const PY_BIN_DIR: &str = "bin";
#[cfg(target_family = "windows")]
pub const PY_BIN_DIR: &str = "Scripts";

File: src/pyenv/consts.py
import platform
from pathlib import Path

PY_ENV_PATH = Path.home() / ".pyenvs"
PY_ENV_PATH.mkdir(exist_ok=True)

PY_VENV_CFG = "pyvenv.cfg"

IS_WINDOWS = platform.system() == "Windows"

if IS_WINDOWS:
    PY_BIN_DIR = "Scripts"
else:
    PY_BIN_DIR = "bin"
